contracts/external/gmxV2/GmxV2IsolationModeTokenVaultV1.sol

```diff
@@ -0,0 +1,421 @@
+import { IGmxExchangeRouter } from "../interfaces/gmx/IGmxExchangeRouter.sol";
+import { IGmxRegistryV2 } from "./GmxRegistryV2.sol";
+import { IGmxV2IsolationModeVaultFactory } from "../interfaces/gmx/IGmxV2IsolationModeVaultFactory.sol";
+import { IIsolationModeTokenVaultV1 } from "../interfaces/IIsolationModeTokenVaultV1.sol";
+
+import { Deposit } from "../interfaces/gmx/GmxDeposit.sol";
+import { EventUtils } from "../interfaces/gmx/GmxEventUtils.sol";
+import { Withdrawal } from "../interfaces/gmx/GmxWithdrawal.sol";
+import { IGmxDepositCallbackReceiver } from "../interfaces/gmx/IGmxDepositCallbackReceiver.sol";
+import { IGmxWithdrawalCallbackReceiver } from "../interfaces/gmx/IGmxWithdrawalCallbackReceiver.sol";
+
+
+/**
+ * @title   GmxV2IsolationModeTokenVaultV1
+ * @author  Dolomite
+ *
+ * @notice  Implementation (for an upgradeable proxy) for a per-user vault that holds the 
+ *          Eth-Usdc GMX Market token that can be used to credit a user's Dolomite balance.
+ */
+contract GmxV2IsolationModeTokenVaultV1 is IsolationModeTokenVaultV1, ProxyContractHelpers {
+    using SafeERC20 for IERC20;
+
+    // ==================================================================
+    // =========================== Constants ============================
+    // ==================================================================
+
+    bytes32 private constant _FILE = "GmxV2IsolationModeVaultV1";
+    bytes32 private constant _VIRTUAL_BALANCE_SLOT = bytes32(uint256(keccak256("eip1967.proxy.virtualBalance")) - 1);
+    bytes32 private constant _VAULT_FROZEN_SLOT = bytes32(uint256(keccak256("eip1967.proxy.vaultFrozen")) - 1);
+    bytes32 private constant _SOURCE_IS_WRAPPER_SLOT = bytes32(uint256(keccak256("eip1967.proxy.sourceIsWrapper")) - 1);
+    bytes32 private constant _SHOULD_SKIP_TRANSFER_SLOT = bytes32(uint256(keccak256("eip1967.proxy.shouldSkipTransfer")) - 1); // solhint-disable max-line-length 
+
+    // ===================================================
+    // ==================== Modifiers ====================
+    // ===================================================
+
+    modifier requireNotFrozen() {
+        Require.that(
+            !isVaultFrozen(),
+            _FILE,
+            "Vault is frozen"
+        );
+        _;
+    }
+    
+    // ==================================================================
+    // ======================== Public Functions ========================
+    // ==================================================================
+
+    function initiateWrapping(
+        uint256 _tradeAccountNumber,
+        uint256[] calldata _marketIdsPath,
+        uint256 _inputAmountWei,
+        uint256 _minOutputAmountWei,
+        IGenericTraderProxyV1.TraderParam[] memory _tradersPath,
+        IDolomiteMargin.AccountInfo[] memory _makerAccounts,
+        IGenericTraderProxyV1.UserConfig memory _userConfig
+    ) external payable nonReentrant onlyVaultOwner(msg.sender) requireNotFrozen() {
```

Comment:
Add new lines to break apart all of these modifiers.

`requireNotFrozen` does not need `()`